src/main/kotlin/no/nav/klage/oppgave/clients/gosys/OppgaveClient.kt

```diff
@@ -130,13 +133,13 @@ class OppgaveClient(
         return uri
     }
 
-    private fun mapType(type: String): String {
+    private fun mapType(type: Sakstype): String {
         return when (type) {
-            TYPE_KLAGE -> BEHANDLINGSTYPE_KLAGE
-            TYPE_FEILUTBETALING -> BEHANDLINGSTYPE_FEILUTBETALING
+            Sakstype.KLAGE -> BEHANDLINGSTYPE_KLAGE
+            Sakstype.FEILUTBETALING -> BEHANDLINGSTYPE_FEILUTBETALING
             else -> {
                 logger.warn("Unknown type: {}", type)
-                type
+                BEHANDLINGSTYPE_KLAGE
```

Comment:
Litt usikker på denne, da man søker på noe man ikke bedt om nødvendigvis?

Reply:
Denne la jeg inn først og fremst for ikke å knekke koden. Søket mot Oppgave er ikke lenger i bruk, så det bør vel kanskje bare fjernes.